test/v1/fixtures/ListChargesFixture.scala

```diff
@@ -24,7 +24,7 @@ object ListChargesFixture {
   val fullDesChargeResponse: JsValue = Json.parse(
```

Comment:
minimalDesChargeResponse and fullDesListChargesSingleResponse can be removed here as they are not currently being used